### DISCUSSION TITLE
PhotoDemon 2024.12

### DISCUSSION THREAD

**Discussion by tannerhelland** [tannerhelland/PhotoDemon#604] [News]
**PhotoDemon 2024.12 is now available.**

Read the [release announcement at photodemon.org](https://photodemon.org/2024/12/05/photodemon-2024-12.html).

Zip file checksums:

`CRC-32: 258c46f3`
`MD5: bfbf7ed762143ca654ce0a5a73ac0a20`
`SHA-1: 38a08ea4f9ed06a48516667b8bf213812ab1aa7d`
`SHA-256: 89a405442fa346900348176265718ad874f65e3ad073d76dd5b1d57179afc7bd`

*(PhotoDemon uses [calendar versioning](https://calver.org/).  Version numbers indicate the year and month of release.)*

<hr /><em>This discussion was created from the release <a href='https://github.com/tannerhelland/PhotoDemon/releases/tag/v2024.12'>PhotoDemon 2024.12</a>.</em>

**Comment by lintalist** [comment]
⌨ Great to see the new feature with the configurable hotkeys!
